include/bitcoin/client/obelisk/obelisk_types.hpp

```diff
@@ -48,8 +40,10 @@ typedef std::vector<history_row> history_list;
 
 struct BCC_API stealth_row
 {
-    data_chunk ephemkey;
-    payment_address address;
+    // No sign byte in public key.
+    hash_digest ephemkey;
+    // No version byte in address.
+    short_hash address;
     hash_digest transaction_hash;
```

Comment:
I mean right now we're using stealth address for payments to 1 person but in the future you might have stealth-multisig addresses. So we cannot assume that the derived address is a normal BTC address- it could be a multisig address. For now it would be a correct assumption, but it won't be.